src/components/ArticleList.jsx

```diff
@@ -1,17 +1,29 @@
-import {React, useState} from 'react'
+import { React, useState } from 'react';
 
 const ArticleList = props => {
-    const [articles, setArticles] = useState([])
-  return (
+    return (
+        <div>
+            <ul className='article-list'>
+                {props.articles.map(article => {
+                    return (
+                        <li className='article-list-item' key={article.article_id}>
+                            <h2>{article.title}</h2>
+                            <p>Topic: {article.topic}</p>
+                            <img
+                                src={article.article_img_url}
+                                alt={article.title}
+                            />
+                            <p>{article.body}</p>
+                            <p>Author: {article.author}</p>
+                            <p>Created at: {article.created_at}</p>
```

Comment:
the created at should be in a more human readable format rather than the timestamp